### PR TITLE
this is the login feature

### PR DESCRIPTION
**Pull request by RoyZone96**
@kaurrevature  can you review the login feature?

### CODE REVIEW

File: login.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Logon: 
+Passwurd:
```
Comment (kaurrevature) [inline login.txt]: @RoyZone96 do you mean password?

**Review by kaurrevature — CHANGES_REQUESTED** [review]
Could you fix that?

**Comment by RoyZone96** [comment]
I fixed the login feature.  One more review should be good.

**Review by RoyZone96 — COMMENTED** [review]
Can you check now?

**Review by kaurrevature — APPROVED** [review]
LGTM

**Comment by kaurrevature** [comment]
Good job @RoyZone96

**Pull request merged**